libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -723,7 +722,7 @@ internal object MapboxRouteLineUtils {
     ) -> RouteLineGranularDistances? =
         { route: NavigationRoute ->
             val points = routePointsProvider(route)
-            if (points?.flatList?.isNotEmpty() == true) {
+            if (points?.stepPoints?.isNotEmpty() == true) {
```

Comment:
It's not exactly the same: stepPoints could be sth like: [[], []] which means they are not empty while flatLost would be empty.

Reply:
Yea, maybe it's better to refactor the receiving function to accept nullable and empty collections.